tools/experiments/experiments.js

```diff
@@ -51,6 +55,7 @@ let ExperimentDef;
  */
 const CANARY_EXPERIMENT_ID = 'dev-channel';
 const RC_EXPERIMENT_ID = 'rc-channel';
```

Comment:
These two lines should eventually be changed to:

```js
const EXPERIMENTAL_CHANNEL_ID = 'experimental-channel';
const BETA_CHANNEL_ID = 'beta-channel';
```

If it's not easy here, separate PR is okay.

Reply:
Done